engine/src/main/java/org/destinationsol/game/asteroid/Asteroid.java

```diff
@@ -182,14 +182,14 @@ private void maybeSplit(SolGame game) {
         float sclSum = 0;
         while (sclSum < .7f * size * size) {
             float speedAngle = SolRandom.randomFloat(180);
-            Vector2 speed = new Vector2();
-            SolMath.fromAl(speed, speedAngle, SolRandom.randomFloat(0, .5f) * MAX_SPLIT_SPD);
-            speed.add(speed);
+            Vector2 splittedSpeed = new Vector2();
```

Comment:
`splitSpeed` not `splittedSpeed` to make more gramatical sense.

Reply:
`splitSpeed` would be the speed at which the speed occurred, whereas this is the speed the spitted parts will have.

Reply:
My problem with that is that 'splitted' is not actually a word. I think split fits in both cases here. Blame the English language :P

_Sent from my Nexus 5X using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

Reply:
Nich is correct on this one - the past and past participle of split is the same.

Blame english :D